libs/common/src/helpers/date.test.ts

```diff
@@ -91,4 +91,73 @@ describe('date helpers', () => {
       expect(result).toEqual(expected);
     });
   });
+
+  describe('addYear', () => {
+    it('should add one year provided date object', () => {
+      const year = 1;
+      const date = new Date('20-09-1989');
+      const oneYear = date.getFullYear() + year;
+
+      const result = addYear(year, date);
+
+      expect(result.getFullYear()).toBe(oneYear);
+    });
+
+    it('should add ten years provided date object', () => {
+      const year = 10;
+      const date = new Date();
+      const tenYears = date.getFullYear() + year;
+
+      const result = addYear(year, date);
+
+      expect(result.getFullYear()).toBe(tenYears);
+    });
+
+    it('should add two years to now', () => {
+      const year = 2;
+      const date = now();
+      const twoYears = date.getFullYear() + year;
```

Comment:
I think the names for the consts are a tad confusing, could it instead be: 

const yearsToAdd = 2
const currentDate = now()
const twoYearsInFutureYear = date.getFullYear() + year

We could also have it in a for.each loop where we test the multiple test cases with different years (e.g. 1, 2 and 10) if we want to be more succinct?